### PR TITLE
Add terraform binary to admin-builder image

### PR DESCRIPTION
**Pull request by JoelSpeed**
This seems like a reasonable tool to add to the admin image (plus I kinda need it for a repository)

### CODE REVIEW

**Review by wonderhoss — DISMISSED** [review]
Good catch

File: images/admin-builder/Dockerfile [hunk 1]
```diff
@@ -45,5 +45,11 @@ RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cl
 ENV PATH=/workspace/google-cloud-sdk/bin:${PATH}
 RUN gcloud --version
 
+# Install Terraform CLI & check it was installed
+ARG TF_VERSION="0.12.2"
```
Comment (mthssdrbrg) [inline images/admin-builder/Dockerfile]: Not entirely sure what the Terraform binary is supposed to be used for, but if it's for validating/formatting of our stuff will `0.12.x` work? IIRC there are some providers (Vault) that don't work on `0.12.x`?

Comment (JoelSpeed) [reply images/admin-builder/Dockerfile]: You raise a very good point! It is exactly for `fmt` and `validate` so let me verify it doesn't work and I'll get back to you. I vaguely remember this came up a couple of weeks ago right?

Comment (JoelSpeed) [reply images/admin-builder/Dockerfile]: 0.12 is causing pain, going to 0.11.14

**Comment by JoelSpeed** [comment]
This has now been tested and is working!

**Review by wonderhoss — APPROVED** [review]

**Pull request merged**